### PR TITLE
Order history

### PR DESCRIPTION
**Pull request by RetroSabian**

### CODE REVIEW

File: src/Components/OrderHistory/OrderDetailComponent.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        headerTintColor: '#fff',
+        headerTitleStyle: {
+            fontWeight: 'bold',
+        },
+    };
+
+    setTableItems({ item } ){
+        return <View style={styles.container}>
+            {
+                this.props.productState.productDetailState && this.props.productState.productDetailState.length > length &&
```
Comment (GeromeSchutte) [inline src/Components/OrderHistory/OrderDetailComponent.js]: its not clear from this variable's name what it represents

File: src/Components/OrderHistory/OrderDetailComponent.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+            </View>
+            <View>
+                <View style={styles.productSummaryContainer}>
+                    <Text style={styles.product}>{item.name}</Text>
+                    <Text>{item.description}</Text>
+                    <TextMask style = {styles.boldText}
+                        value= {parseFloat(item.price)}
+                        type={'money'}
+                        options={{
+                            precision: 2,
```
Comment (GeromeSchutte) [inline src/Components/OrderHistory/OrderDetailComponent.js]: possible that we could separate textmask's options into a formatting constant somewhere? i see it is used all over.

File: src/Components/OrderHistory/OrderDetailComponent.js [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+                </View>   
+            </View>
+        </View>;
+    }
+
+    render() {
+        return ( 
+            <ScrollView>
+                <FlatList 
+                    data = {this.props.orderHistoryState.orderHistory[this.props.orderHistoryState.index].line_items}
```
Comment (GeromeSchutte) [inline src/Components/OrderHistory/OrderDetailComponent.js]: please make your prop assignment spacing consistent.

File: src/Components/OrderHistory/SubViews/OrderHistoryListComponent.js [hunk 4]
```diff
@@ -4,60 +4,62 @@ import { View,
...
-                </View>
-                <View style= {styles.row}>
-                    <Text style= {styles.itemName}>Status:</Text>
-                    <Text style= {styles.itemName}>{item.state}</Text>
-                </View>
-                <View style= {styles.row}>
-                    <Text style= {styles.itemName}>Special instructions:</Text>
-                    <Text style= {styles.itemName}>{item.instructions}</Text>
+    setTableItems({ item, index } ){
+        return <TouchableWithoutFeedback onPress ={ () => this.setIndex(index)}>
```
Comment (GeromeSchutte) [inline src/Components/OrderHistory/SubViews/OrderHistoryListComponent.js]: how did you manage to use 3 different types of prop assignment spacing in the same component

File: src/Components/OrderHistory/SubViews/OrderHistoryListComponent.js [hunk 5]
```diff
@@ -4,60 +4,62 @@ import { View,
...
                 </View>
-                <View style= {styles.row}>
-                    <Text style= {styles.itemName}>Date Complete:</Text>
-                    <Text style= {styles.itemName}>{item.completed_at.substring(firstIndex, eleventhIndex)}</Text>
-                </View>
-                <Text style= {styles.itemName}>Click to examine items</Text>            
+                <TextMask style = {styles.itemPrice}
+                    value= {item.price}
+                    type={'money'}
+                    options={{
```
Comment (GeromeSchutte) [inline src/Components/OrderHistory/SubViews/OrderHistoryListComponent.js]: formatting constaaant pls

File: src/Components/OrderHistory/OrderDetailComponent.js (hunk 1 above)
Comment (RetroSabian) [reply src/Components/OrderHistory/OrderDetailComponent.js]: made variable zero

File: src/Components/OrderHistory/SubViews/OrderHistoryListComponent.js (hunk 4 above)
Comment (RetroSabian) [reply src/Components/OrderHistory/SubViews/OrderHistoryListComponent.js]: what do you mean?

File: src/Components/OrderHistory/SubViews/OrderHistoryListComponent.js (hunk 5 above)
Comment (RetroSabian) [reply src/Components/OrderHistory/SubViews/OrderHistoryListComponent.js]: done throughout

File: src/Components/OrderHistory/OrderDetailComponent.js (hunk 2 above)
Comment (RetroSabian) [reply src/Components/OrderHistory/OrderDetailComponent.js]: done throughout

File: src/Components/OrderHistory/SubViews/OrderHistoryListComponent.js (hunk 4 above)
Comment (RetroSabian) [reply src/Components/OrderHistory/SubViews/OrderHistoryListComponent.js]: okay so i checked through the files spacing, and think I did it to a standard

File: src/Components/Home/LandingComponent.js [hunk 6]
```diff
@@ -42,6 +43,17 @@ class LandingComponent extends Component {
         }
     }
 
+    componentWillMount(){
+        const { productActions } = this.props;        
+        const vendorParams = { serviceProviderId: '1',
+            match: this.props.loginState.userId };
+        let jwt = this.props.loginState.jwt;
```
Comment (GeromeSchutte) [inline src/Components/Home/LandingComponent.js]: not crazy about this. think the function should take both jwts and just use whichever is applicable, to make its use consistent

**Pull request merged**